ReSharper.FSharp/ReSharper.FSharp.sln.DotSettings.user

```diff
@@ -0,0 +1,8 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
And this was the not needed one. :)
(Please vote for [RSRP-478535](https://youtrack.jetbrains.com/issue/RSRP-478535), cc @estrizhok)